app/color

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
Good naming - the filename reflects the contents (color definitions) :+1: 
Any reason this file does not have the `js` extension?
Without that extension, some viewers may not show code-highlights associated with JavaScript